.ci/azure-pipelines/azure-pipelines.yaml

```diff
@@ -14,29 +16,113 @@ stages:
     displayName: Formatting
     jobs:
       - template: formatting.yaml
-  - stage: build_ubuntu
-    displayName: Build Ubuntu
+
+  - stage: build_gcc
+    displayName: Build GCC
```

Comment:
```suggestion
    displayName: GCC
```

Reply:
Do this for the rest too?

Reply:
I think we should keep the name `Build GCC`  since it implies it is the build stage in the pipeline.

Reply:
If you look at the synopsis shown, it seems redundant.  But if you look at the [extended information](https://dev.azure.com/PointCloudLibrary/pcl/_build/results?buildId=16460&view=results), it seems necessary.

<img width="433" alt="Screen Shot 2020-05-25 at 8 06 06" src="https://user-images.githubusercontent.com/2657068/82766982-a7f22480-9e5e-11ea-881a-16a105e43fbe.png">

Reply:
The Github one can be changed I think if we properly integrate with Github workflow.
[Here](https://github.com/github/VisualStudio/pull/2512/checks?check_run_id=653837445) they have seemed to integrate properly with workflow. 
Changing it also might create problems if we have a separate stage for text in the future 

It's not a big issue anyway will leave finally call to you.

Reply:
> It's not a big issue anyway

Yeah. Plus that example is of GitHub actions. Something very different (and quite similar)